### ISSUE TITLE
Container image ghcr.io/interlynk-io/sbomqs doesn't run in GitLab CI/CD job 

### ISSUE THREAD

**Issue by rutger-gerritsen** [interlynk-io/sbomqs#238]
Container image ghcr.io/interlynk-io/sbomqs doesn't run in GitLab CI/CD job. It doesn't meet the minimal image requirements to run it.  

> Image requirements
Any image used to run a CI/CD job must have the following applications installed: sh or bash and grep

It would be really nice if sbomqs image container supports this so it can be used in a GitLab CI/CD pipeline.

- https://github.com/interlynk-io/sbomqs/blob/main/Dockerfile#L12
- https://docs.gitlab.com/ee/ci/docker/using_docker_images.html#image-requirements

**Comment by riteshnoronha** [comment]
@rutger-gerritsen  a fix is here https://github.com/interlynk-io/sbomqs/pull/239 however i have no way to test it out in gitlab. If you can test it and let me know it works, then i can merge it.

**Comment by rutger-gerritsen** [comment]
> @rutger-gerritsen a fix is here #239 however i have no way to test it out in gitlab. If you can test it and let me know it works, then i can merge it.

@riteshnoronha the fix didn't work yet in our GitLab. I got it working in our GitLab by changing the following lines in the Dockerfile.

```
COPY --from=builder /bin/bash /bin/bash
COPY --from=builder /bin/grep /bin/grep
```
TO
```
COPY --from=builder /bin/sh /bin/grep /bin/busybox /bin/touch /bin/chmod /bin/mkdir /bin/date /bin/
```

So `sh` instead of `bash` and some additional applications.

**Comment by riteshnoronha** [comment]
Incorporated this here https://github.com/interlynk-io/sbomqs/pull/240

**Comment by riteshnoronha** [comment]
Ok merged. Do let us know youre use case would love to learn more about it.

**Comment by rutger-gerritsen** [comment]
@riteshnoronha Thanks for merging the changes :-) Hopefully you can published it soon on ghcr.io/interlynk-io/sbomqs?

**Comment by riteshnoronha** [comment]
@rutger-gerritsen yes i will create a release this friday. 

This is a great use case. We have recently added compliance  reports in sbomqs, we think CRA might eventually adopt BSI TR-03183 for sboms, so one of the first compliance reports we added is [TR-03183](https://github.com/interlynk-io/sbomqs?tab=readme-ov-file#compliance-reports).  We plan to add a couple more compliance reports soon (NTIA minimum/OSS license/SCVS etc).

If ever your org would like to enrich sboms(eol/eos/purl/cpe) or distribute them, do keep us in mind, we are a great solution for it. Here is one way we have helping OSS projects, manage and display their health https://github.com/interlynk-io/sbomqs?tab=readme-ov-file#sbom-card.

As always if u have any feature requests or bugs, do let us know we address them.
